### ISSUE TITLE
report is not nice

### ISSUE THREAD

**Issue by 1muen** [artilleryio/artillery#850]
![grafik](https://user-images.githubusercontent.com/49640488/85864851-56261c80-b7c5-11ea-93b6-cbd5a5e831ba.png)

- [ ] The Y-Axis description intersects with the numbers
- [ ] Add lines between the sections, so it will be easier to understand, which graph belongs to which headline
- [ ] The X-Axis and Y-axis are teared appart, and the X-axis doesnt start at 0

**Comment by megamorf** [comment]
I agree, here's an example from I report I just generated. It's needlessly hard to decipher:
![image](https://user-images.githubusercontent.com/1732128/96139287-fe8bf800-0efe-11eb-9f2a-0717730348b9.png)

**Comment by ChrisKatsaras** [comment]
I agree with you two. I think the reason for this is simply that the graphs were developed so long ago, to meet the base requirements. However, now that the framework has expanded, the visualizations could be updated to provide a richer user experience
